CorsixTH/Lua/dialogs/resizables/options.lua

```diff
@@ -78,7 +79,7 @@ end
 
 
 function UIOptions:UIOptions(ui, mode)
-  self:UIResizable(ui, 320, 500, col_bg)
+  self:UIResizable(ui, 620, 300, col_bg)
```

Comment:
The width is very close to the min default of 640. Does it look okay in 640x480?

Reply:
<img width="752" alt="Screenshot 2024-01-20 at 16 38 53" src="https://github.com/CorsixTH/CorsixTH/assets/552285/54759df3-4a22-4d9d-aa70-0ff503b3c494">

I think that's ok, it's better than the toggle labels and buttons in one column and the four regular buttons in a second column.